README.md

```diff
@@ -12,7 +12,6 @@ single Transformer model on hundreds of computers from different universities, c
 
 ## Key Features
 
-* Train neural networks of arbitrary size: parts of their layers are distributed across the participants.
```

Comment:
We can describe MoE somewhere in readme, but this sentence here was totally misleading